### ISSUE TITLE
Louder volume

### ISSUE THREAD

**Issue by GoKamiSo4476** [sudachi-emu/sudachi#97]
Since the original yuzu volume was kinda small, can you make it louder like going 120 or 130% of output volume

**Comment by DZXTRICKS** [comment]
Agree, most games are sounding tiny and have to resort on very high device volume
